Library/Homebrew/brew.rb

```diff
@@ -144,8 +144,8 @@ def require?(path)
   Utils::Analytics.report_exception(e)
   onoe e
   if internal_cmd && defined?(OS::ISSUES_URL)
-    $stderr.puts "#{Tty.white}Please report this bug:"
-    $stderr.puts "    #{Tty.em}#{OS::ISSUES_URL}#{Tty.reset}"
+    $stderr.puts "Please report this bug:"
+    $stderr.puts "    #{Formatter.url(OS::ISSUES_URL)}"
```

Comment:
I think I'd like to preserve the boldface in the first line (`#{Tty.bold}Please report this bug:#{Tty.reset}`) and, since we're touching this anyway, adjust the indentation of the second line to two spaces which better aligns with most (if not all) similar user-visible output.